app/routes/docs.contributing.github.mdx

```diff
@@ -0,0 +1,49 @@
+# GitHub
+
+For all projects in the [nasa-gcn](https://github.com/nasa-gcn) GitHub organization, contributors should adhere to the following steps:
+
+- Fork a copy of our repo
+
+- Clone your repo to your machine
+
+- Create a branch for your new changes
+
+- Create a Pull Request to contribute to the original project
+
+For more information, check out our [web-dev-bootcamp](https://github.com/nasa-gcn/web-dev-bootcamp) repository. In the syllabus folder you will find some instructive lectures and examples to help understand our workflow.
+
+## Forking a Repo
+
+Navigate to the page repository page for whichever project you wish to contribute to, for example: [gcn.nasa.gov](https://github.com/nasa-gcn/gcn.nasa.gov)
+
+Click the Fork button
+
+Click Create Fork to create a new repo in your account
+
+## Cloning a Repo
+
+With your new fork now created, click the Code button and copy the URL.
```

Comment:
do we need to put anything about ssh keys and agents before this point as they won't be able to clone it with ssh otherwise or do we assume they will figure that part out on their own? this is really good github documentation for how to do that https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account

Reply:
I wasn't too sure how much info to drop here, but this would be useful. I can add a line that includes the link